lib/options.dart

```diff
@@ -50,79 +76,95 @@ class UiOptions {
       'orientation': orientation?.stringValue,
       'activity_compatibility_request_code': activityCompatibilityRequestCode,
       'close_button_tint_color': closeButtonTintColor?.hex,
+      if (logoImage != null) 'logo_image': base64.encode(encodePng(logoImage!)),
```

Comment:
This is the **collection-if** operator. Can be used to only add a key-value pair to a map if a certain condition is true.